MyMovies/Views/MovieTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MovieTableViewCell.swift
+//  MyMovies
+//
+//  Created by Marc Jacques on 5/3/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class MovieTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var addButton: UIButton!
+    
+    var movieController: MovieController?
+    var searchedMovie: MovieRepresentation? {
+        didSet {
+            updateViews()
+        }
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
```

Comment:
Make sure to get rid of these comments or chunks of code if you are not using it.
If it is an important comment then keep it, but not in this case.